.github/workflows/ci.yaml

```diff
@@ -287,3 +287,47 @@ jobs:
             scripts/images/**/flatcar_production_*.sh
             scripts/images/**/flatcar_test_update.gz
             scripts/ebuild_logs.tar.xz
+  test:
+    name: "Run kola tests"
+    needs: packages
+    runs-on:
+      - debian
+      - kola
+      - self-hosted
+      - ${{ matrix.arch }}
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: ["amd64", "arm64"]
+    defaults:
+      run:
+        working-directory: scripts
+
+    steps:
```

Comment:
Considering this is a new job, running on a fresh runner, you will need to also clone the repo, check out submodules, etc:

https://github.com/flatcar-linux/scripts/blob/main/.github/workflows/ci.yaml#L77-L81

This might also be of interest in case some submodules are from different remotes:

https://github.com/flatcar-linux/scripts/blob/main/.github/workflows/ci.yaml#L109-L130

Reply:
Although, I am not sure if the submodules are needed to run kola tests. You can probably skip updating remotes and pulling the refs. Just the stuff from the first link should be enough  (i think).

Reply:
*sigh*

The runners are vanilla. So you'll have to install the needed bits for now:

https://github.com/flatcar-linux/scripts/blob/main/.github/workflows/ci.yaml#L58-L75

You can pick an choose which binaries make sense and add any additional packages the tests need to run.